temporian/core/operators/select.py

```diff
@@ -28,8 +28,15 @@ def __init__(self, event: Event, feature_names: Union[str, List[str]]):
         super().__init__()
 
         # store selected feature names
-        if isinstance(feature_names, str):
+        if isinstance(feature_names, list):
+            pass
+        elif isinstance(feature_names, str):
             feature_names = [feature_names]
+        else:
```

Comment:
nit, subjective, this would make it cleaner, avoiding the if/pass clause:
```
if (isinstance(feature_names, str):
    feature_names = [feature_names]
    
if not isinstance(feature_names, list):
    raise ...
```